pages/fail-whale.vue

```diff
@@ -73,4 +87,21 @@ export default {
       }
     }
   }
+
+  hr {
+    text-align: center;
+    margin-top: 18px;
+    margin-bottom: 18px;
+    max-width: 450px;
+  }
+
+  hr:after {
+    background: var(--body-bg);
+    color: var(--body-text);
+    content: 'or';
```

Comment:
'or' should be localized - you'll need to switch that from being rendered in the :after to do that